src/transformers/models/encoder_decoder/modeling_encoder_decoder.py

```diff
@@ -458,6 +458,33 @@ def prepare_inputs_for_generation(
         }
         return input_dict
 
+    def resize_token_embeddings(
+        self,
+        new_num_tokens_encoder: Optional[int] = None,
+        new_num_tokens_decoder: Optional[int] = None
```

Comment:
I'm not sure if we really need this method. Here encoder and decoder are essentially different models so their embedding can be resized by calling the `resize_token_embeddings` method on the respective models. Maybe just throw an error here with a message that resizing should be done using the individual models.

cc @patrickvonplaten

Reply:
I can change that if you want but something should be done because the method is already there and throwing an undocumented error at the moment. Throwing an error or keeping the current implementation of this PR is fine for me. Since you cc'd Patrick, I will wait for his reply before changing anything.

Reply:
Agree with both of you and would propose to overwrite the method just to give a better error.

```python
def resize_token_embeddings(...):
   # throw nice warning to tell the user to use model.encoder.resize_token_embeddings(...) and model.decoder.resize...
```